bin/pping.py

```diff
@@ -206,15 +203,16 @@ def help():
     -v  --version   Display version and exit
 
     Written by Stian Søiland and Magnus Nordseth, 2002
```

Comment:
Ooh, our policy is to not leave author names around in single files. That's what commitlogs are for. This could easily be removed too :)